.craft.yml

```diff
@@ -9,7 +9,9 @@ statusProvider:
   name: github
   config:
     contexts:
-      - 'Trigger: 7973c658-e248-4a14-8552-7a66caa7fc0c'
+      - 'onpremise-builder (sentryio)'
```

Comment:
I see `onpremise-github-pr-check (sentryio)` on this PR, is it the same check?

Reply:
Nope, they are different. One is specific to pull requests, the one we wait on here is for pushes:

![image](https://user-images.githubusercontent.com/126780/91462222-9e7bbc80-e892-11ea-99fb-2586b48089f7.png)


(good catch tho)

Reply:
Cool 👍